src/components/ErrorBoundary/index.tsx

```diff
@@ -84,7 +84,7 @@ const propTypes = {
 }
 
 const defaultProps = {
-  Wrapper: ({ children }) => <>{children}</>,
+  Wrapper: <></>,
```

Comment:
I found that a default value of a functional component instead of plain JSX gave an error in Storybook

Reply:
Nevermind, using the plain JSX tag gives an error with the testing library